### DISCUSSION TITLE
Vray Not Support Some Game

### DISCUSSION THREAD

**Discussion by nyaruru1** [XTLS/Xray-core#3523] [Q&A]
How to used v2Ray on Minecraft Bedrock

I connect to v2ray & trying to play Minecraft bedrock server but game say please check connect

But after I disconnect v2ray my server is normal

Minecraft server has UDP Protocol 19132

I used vless+ws

**Comment by nyaruru1** [comment]
Pls Help me

**Comment by mmmray** [comment]
this should not be filed as an issue with xray, if it is a bug then there is a lot of information missing from the bugreport. i have converted it to a discussion.

some questions:

* which client do you use
* does anything UDP work at all
* is it only one config that has this issue

**Reply by nyaruru1** [reply]
Client
V2Box, Netmod

UDP not work in port 19132

**Comment by nyaruru1** [comment]
[config (2).json](https://github.com/user-attachments/files/16162793/config.2.json)


This a Config file I connect to port 80 & play game with udp server not work

**Reply by nyaruru1** [reply]
You can help me?
